### PR TITLE
[Publisher] Update Help Center Guide for New Settings Page

### LINKED ISSUE #1337: [Publisher] Update screenshots and guide for Agency Settings page

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Update copy, links and screenshots within the Help Center Guide for new Settings page.

Screenshot 2024-05-22 at 9 14 17 AM
Screenshot 2024-05-22 at 9 15 40 AM

## Related issues

Closes #1337

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by helperbot-recidiviz** [comment]
@mxosman successfully triggered a playtest deployment. Full deployment usually takes 5 minutes. Your playtest link is https://mahmoudtest---publisher-web-b47yvyxs3q-uc.a.run.app/

File: publisher/src/components/HelpCenter/Guides/SettingsGuide.tsx [hunk 1]
```diff
@@ -67,8 +64,8 @@ export const AgencySettingsGuide = () => (
     <Styled.SectionWrapper>
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SettingsGuide.tsx:64]: nit can we change [line 56 above](https://github.com/Recidiviz/justice-counts/pull/1338/files#diff-4122ee1d4a52e5a07d094a883fa7dd6fe9798daa4099fc7ea6649624d8f809e4L59) to "When setting jurisdictions"

**Review by morden35 — APPROVED** [review]
thank you Mahmoud! looks fantastic!

File: publisher/src/components/HelpCenter/Guides/SettingsGuide.tsx (hunk 1 above)
Comment (mxosman) [reply publisher/src/components/HelpCenter/Guides/SettingsGuide.tsx:64]: Oo great eye! Thank you, Michelle!

**Comment by mxosman** [comment]
Thank you for the lightning fast reviews, Michelle!!

**Pull request merged**